api/client/samples/analogous_years/run_analogous_years.py

```diff
@@ -111,6 +111,8 @@ def main():
     parser.add_argument('--all_ranks', action='store_true', help='Lets you see all the ranks'
                                                                  'as opposed to separate method'
                                                                  'ranks')
+    parser.add_argument('--num_jobs', type=int, default=0,
```

Comment:
shouldn't the default be 1? you cant run on zero threads ;)

Reply:
https://github.com/blue-yonder/tsfresh/issues/458

Reply:
Also, celery no longer complaining and sending out emails.

Reply:
just double checked the doc for tsfresh https://tsfresh.readthedocs.io/en/latest/api/tsfresh.feature_extraction.html#tsfresh.feature_extraction.extraction.extract_features
`n_jobs (int) – The number of processes to use for parallelization. If zero, no parallelization is used.` So we should be good with 0.